protocol/v2/ssv/runner/committee.go

```diff
@@ -230,12 +231,13 @@ func (cr *CommitteeRunner) ProcessConsensus(ctx context.Context, logger *zap.Log
 		}
 		switch duty.Type {
 		case spectypes.BNRoleAttester:
-			validDuties++
 			attestationData := constructAttestationData(beaconVote, duty)
 			partialMsg, err := cr.BaseRunner.signBeaconObject(cr, duty, attestationData, duty.DutySlot(),
 				spectypes.DomainAttester)
 			if err != nil {
-				return errors.Wrap(err, "failed signing attestation data")
+				logger.Error("failed signing attestation data for duty",
```

Comment:
if we know this can happen maybe its a warning? we don't want to freak out operators..

Reply:
I guess more opinions will be appriciated @moshe-blox @MatusKysel @andrew-blox